docs/guides/getting-started/setup/integrate.mdx

```diff
@@ -0,0 +1,113 @@
+---
+description: If you already have an existing web project, this guide will walk you through integrating Tauri into your project
+pagination_next: guides/development/development-cycle
+sidebar_position: 3
+sidebar_custom_props:
+  doc_card_image: /img/guides/getting-started/setup/html5-light.svg
+  doc_card_image_dark: /img/guides/getting-started/setup/html5-dark.svg
+---
+
+import Commands from './_fragments/_commands.mdx'
+import TauriInit from './_fragments/_tauri-init.mdx'
+import { InstallTauriCli, InstallTauriApi } from '@theme/Command'
+
+# Integrate into Existing Project
+
+If you already have an existing web project, this guide will walk you through integrating Tauri into your project, whether it is Node.js-based (like [Svelte], [React], [Vue], or [Angular]) or Rust-based (like [Yew] or [Dominator]).
+
+:::info
+
+Before we continue, make sure you have completed the [prerequisites] to have a working development environment.
+
+:::
+
+Although Tauri is compatiable with nearly any frontend framework, we'll use examples with a [React] project throughout this guide. We'll assuming you're starting off with a project structure similar to this:
+
+```
+.
+│── package.json
+│── public
+│   ╰── index.html
+╰── src
+    │── App.css
+    │── App.js
```

Comment:
I think react uses .jsx 
```suggestion
    │── App.jsx
```

Reply:
You can use either with React, adding `x` allows you to use XML-style tags inside your JS from what I understand. But will take the x here